src/Employer/Employer.Web/ViewModels/Part1/TrainingProvider/SelectTrainingProviderEditModelValidator.cs

```diff
@@ -1,13 +1,18 @@
 ﻿using Esfa.Recruit.Vacancies.Client.Application.Validation;
 using FluentValidation;
 
-namespace Esfa.Recruit.Employer.Web.ViewModels.Part2.TrainingProvider
+namespace Esfa.Recruit.Employer.Web.ViewModels.Part1.TrainingProvider
 {
     public class SelectTrainingProviderEditModelValidator : AbstractValidator<SelectTrainingProviderEditModel>
     {
         public SelectTrainingProviderEditModelValidator()
         {
-            When(m => m.SelectionType == TrainingProviderSelectionType.Ukprn, () =>
+            RuleFor(m => m.SelectTrainingProvider)
+                .NotNull()
+                .WithMessage("Please select an option to continue");
+
+            When(m => m.SelectTrainingProvider == true && 
```

Comment:
If you rename this property I don't think you will need the `== true`

Reply:
Its a nullable bool so this needs to be explicit